### PR TITLE
Update 1.4.6

### PR DESCRIPTION
**Pull request by Arishamays1**
## ☆ Powerlaw 1.4.6 Update ☆
[Jira Ticket](https://anaconda.atlassian.net/browse/PKG-2458)
[Upstream](https://github.com/jeffalstott/powerlaw)
[Dependencies ](https://github.com/jeffalstott/powerlaw/blob/master/setup.py)
# Changes
- Updated version number and `sha256`
- Updated pinnings on dependencies
- A bit of reformatting of the `meta.yaml`
- Added `pip` related testing

### CODE REVIEW

**Review by varlackc — APPROVED** [review]
LGTM

File: recipe/meta.yaml [hunk 1]
```diff
@@ -27,14 +28,23 @@ requirements:
...
+  license: MIT
+  license_family: MIT
+  license_file: powerlaw.py
+  description: |
+    powerlaw is a toolbox using the statistical methods developed in
+    Clauset et al. 2007 and Klaus et al. 2011 to determine if a probability
+    distribution fits a power law.
+  summary: Toolbox for testing if a probability distribution fits a power law
+  doc_url: https://github.com/jeffalstott/powerlaw/blob/master/README.rst
+  dev_url: https://www.github.com/jeffalstott/powerlaw
```
Comment (boldorider4) [inline recipe/meta.yaml]: can you fix the URL? You should change it to `https://github.com/jeffalstott/powerlaw`so the linter doesn't complain.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -27,14 +28,23 @@ requirements:
...
+    - pip
 
 about:
-  home: "http://www.github.com/jeffalstott/powerlaw"
-  license: "MIT"
-  license_family: "MIT"
-  license_file: "powerlaw.py" 
-  summary: "Toolbox for testing if a probability distribution fits a power law"
-  
+  home: https://www.github.com/jeffalstott/powerlaw
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]:
```suggestion
  home: https://github.com/jeffalstott/powerlaw
```

File: recipe/meta.yaml (hunk 1 above)
Comment (skupr-anaconda) [inline recipe/meta.yaml]:
```suggestion
  dev_url: https://github.com/jeffalstott/powerlaw
```

**Review by skupr-anaconda — APPROVED** [review]
LGTM but `home` and `dev_url`

File: recipe/meta.yaml (hunk 1 above)
Comment (Arishamays1) [reply recipe/meta.yaml]: no problem!

**Review by boldorider4 — APPROVED** [review]

**Pull request merged**